README.md

```diff
@@ -38,3 +45,7 @@
 
 - UISI: Unknown Inbound Session ID
 - UTD: Unable to Decrypt
+
+## V
+
+- VoIP: Voice over Internet Protocol (voice and video calls)
```

Comment:
```suggestion
```

I'd probably remove this one as it is not really related to Matrix, and seems pretty ubitquitous

Reply:
Arguably, so is E2EE, but I guess that VoIP is a bit more known